### ISSUE TITLE
Get unwanted comment with javascript language

### ISSUE THREAD

**Issue by dangelion** [PrismJS/prism#822]
In this snippet of javascript, Prism read the **/*** as the begin of a comment.
```
    watch: {
      less: {
        files: [
          'path/*.css'
        ],
        tasks: ['css:dev']
      }
    }
```
So I see the code commented from there to the end!

**Comment by Golmote** [comment]
Thanks for reporting. This is a known issue, though.

**Comment by dangelion** [comment]
So is still a bug? Have you in plan to solve it?

**Comment by Golmote** [comment]
Yes I'd like to get this solved at some point. But the issue is not that trivial. Several PRs have already been submitted in an attempt to fix it. We are definitely working on fixing this but it might take some time.
I'll keep this issue open for now.

**Comment by Golmote** [comment]
This must have been fixed by #967. I cannot reproduce the issue on the [Test page](http://prismjs.com/test.html) anymore.
